inst/stan/estimate_infections.stan

```diff
@@ -29,6 +29,34 @@ transformed data{
   // Rt
   real r_logmean = log(r_mean^2 / sqrt(r_sd^2 + r_mean^2));
   real r_logsd = sqrt(log(1 + (r_sd^2 / r_mean^2)));
+
+  int delay_max_fixed = (n_fixed_delays == 0 ? 0 :
+    sum(delay_max[fixed_delays]) - num_elements(fixed_delays) + 1);
+  int delay_max_total = (delays == 0 ? 0 :
+    sum(delay_max) - num_elements(delay_max) + 1);
+  vector[gt_fixed[1] ? gt_max[1] : 0] gt_fixed_pmf;
+  vector[truncation && trunc_fixed[1] ? trunc_max[1] : 0] trunc_fixed_pmf;
+  vector[delay_max_fixed] delays_fixed_pmf;
+
+  if (gt_fixed[1]) {
+    gt_fixed_pmf = discretised_pmf(gt_mean_mean[1], gt_sd_mean[1], gt_max[1], gt_dist[1], 1);
+  }
+  if (truncation && trunc_fixed[1]) {
+    trunc_fixed_pmf = discretised_pmf(
```

Comment:
why is no truncation being used here?

Reply:
to allow zero delays (which we can support unlike 0 generation times)

Reply:
which are happening in the indexing already? This is to truncate the mean vs right shifting?

Reply:
I've added a news note indicating this is a breaking change and updated the arg to have a clearer name (`left_truncate`).